backend/xivdpscalc/models/tags.py

```diff
@@ -0,0 +1,37 @@
+"""Model for tags in the database"""
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import relationship, declared_attr
+from sqlalchemy.schema import Column, Table
+from sqlalchemy.types import Integer, String
+
+from xivdpscalc import db
+
+TAG_NAME_MAX_LENGTH = 64
+HEX_COLOR_LENGTH = 7 # #0086BF is 7 characters long
+
+
+class Tag(db.Model):
+  __tablename__ = 'tags'
+
+  id = Column(Integer, primary_key=True)
+  name = Column(String(TAG_NAME_MAX_LENGTH))
+  color = Column(String(HEX_COLOR_LENGTH)) # hex coloring in the UI
```

Comment:
can we store this as a hex value and then provide a utility tostring function if needed?

Reply:
This might require a tiny bit more discussion, but I created a custom type to handle this. Might consider raising some custom exceptions in the new type, but eh.